npm/webpack-preprocessor/test/unit/index.spec.js

```diff
@@ -264,7 +264,7 @@ describe('webpack preprocessor', function () {
           this.compilerApi.plugin.withArgs('compile').yield()
           this.compilerApi.watch.yield(null, this.statsApi)
 
-          return Promise.delay(10) // give assertion time till next tick
+          return Promise.delay(11) // give assertion time till next tick
```

Comment:
this fails for me locally unless it is 11 :/ I don't think its related to my changes...